packages/esm-service-queues-app/translations/en.json

```diff
@@ -275,6 +275,7 @@
   "ticketNumber": "Ticket Number",
   "time": "Time",
   "timeCannotBeInFuture": "Time cannot be in the future",
+  "timeCannotBePriorToPreviousQueueEntry": "Time cannot be before start of previous queue entry: {{time}}",
   "timeOfTransition": "Time of transition",
   "tirageNotYetCompleted": "Triage has not yet been completed",
```

Comment:
```suggestion
  "triageNotYetCompleted": "Triage has not yet been completed",
```

Reply:
Hi @denniskigen!
This change in translation is not related to this PR, the change came up from the `extract-translations` command. Let me get another PR to improve the key name.
Thanks!